### DISCUSSION TITLE
Cross Plattform Ready ?

### DISCUSSION THREAD

**Discussion by markusalbrecht-procam** [lepoco/wpfui#769] [General]
Hi,
I read that this is based on the MS fluent icon system. So the question is as we are developing an AvaloniaUi Desktop App if this can be used then also on Mac OS, Linux,... I assume not as it is based on MS frameworks, right ?
Br
Markus

**Comment by MrCosmo** [comment]
All though .NET is cross platform, Wpf is not.

**Comment by markusalbrecht-procam** [comment]
Thought so - no problem a there is a FluentTheme integration within AvaloniaUI already was just an ides to maybe reuse some parts of this framework too. Thnaks for the response.
